config/develop/namespaced/sqs-dispatch-to-raw.yaml

```diff
@@ -0,0 +1,12 @@
+template:
+  path: sqs-queue.yaml
+parameters:
+  MessageRetentionPeriod: "1209600"
```

Comment:
Odd that these are all strings, but is this an artifact of sceptre?

Reply:
It's [Cloudformation CLI behavior](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html). Parameters must be string, even when they are defined in the template as `Number`, as is the case here.

If I try to use an integer:

```
"Parameter validation failed:\nInvalid type for parameter Parameters[0].ParameterValue, value: 1209600, type: <class 'int'>, valid types: <class 'str'>"
```